rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -358,6 +358,7 @@ static bool load_sigstruct(const char *path, void *sigstruct)
 	return true;
 }
 
+#ifdef DEBUG_ENCLAVE
```

Comment:
This is not right way. The detection of the necessity of launch token **MUST** be dynamic. You change the flexibility in a static way. Plz remove DEBUG_ENCLAVE.